read-fonts/src/tables/variations.rs

```diff
@@ -411,28 +411,35 @@ impl<'a> PackedDeltas<'a> {
     }
 }
 
+/// The type, notably influence how many bytes per entry, of value in a delta run
```

Comment:
I found this hard to grok, maybe just,

```suggestion
/// The type of values for a given delta run (influences the number of bytes per delta)
```

or something?